src/main/java/jp/co/soramitsu/sora/didresolver/controllers/DIDResolverBaseController.java

```diff
@@ -18,25 +19,28 @@
 @RestController
 @RequestMapping(DIDResolverBaseController.PATH)
 @Api(value = DIDResolverBaseController.PATH, description = "CRUD operations on DID documents")
+@Slf4j
 public class DIDResolverBaseController {
 
-    static final String PATH = "/did";
-    static final String ID_PARAM = "/{did}";
+  static final String PATH = "/did";
+  static final String ID_PARAM = "/{did}";
 
-    StorageService storageService;
+  StorageService storageService;
 
-    DIDResolverBaseController(StorageService storageService) {
-        this.storageService = storageService;
-    }
+  DIDResolverBaseController(StorageService storageService) {
+    this.storageService = storageService;
+  }
 
     @PostMapping(consumes = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
it looks like some parts of this file use 2 spaces for indentation and some parts 4 spaces. Please, apply formatter